packages/expo-ui/ios/Modifiers/View+GestureModifiers.swift

```diff
@@ -7,13 +7,15 @@ internal extension View {
   @ViewBuilder
   func applyOnTapGesture(useTapGesture: Bool?, eventDispatcher: EventDispatcher, useContentShape: Bool = false) -> some View {
     if useTapGesture == true {
-      self
-        .if(useContentShape) {
-          $0.contentShape(Rectangle())
-        }
-        .onTapGesture {
-          eventDispatcher()
-        }
+      if #available(iOS 13.0, macOS 10.13, tvOS 16.0, *) {
+        self
+          .if(useContentShape) {
+            $0.contentShape(Rectangle())
+          }
+          .onTapGesture {
```

Comment:
oh no, without `onTapGesture`, people cannot have interaction on the screen. hope we can drop older tvos support soonish

Reply:
Pretty much all Apple TVs should be on 17 or 18 now